### ISSUE TITLE
Austrian NeTEx feed

### ISSUE THREAD

**Issue by langbein-daniel** [public-transport/european-transport-feeds#17]
The daily updated scheduled data of Austria is available here: https://www.mobilitydata.gv.at/en/daten/soll-fahrplandaten-netex

It has a custom license which allows free-of-charge usage: https://www.mobilitydata.gv.at/sites/default/files/metadataset/contract_examples/Lizenzvereinbarung_DBP_v1.1.pdf

**Comment by juliuste** [comment]
Hey, thanks for contributing! 🙂 Unfortunately - if I understood the Austrian data portal correctly - there is no aggregated feed for the entire country yet, instead the site links to multiple feeds for the different regions (see also: [requirements for feeds on this list](https://github.com/public-transport/european-transport-feeds#contributing)). I'll therefore leave this issue open as documentation of the current situation. Please correct me if I misunderstood the situation and there is indeed a combined feed for the entire country 😄

**Comment by langbein-daniel** [comment]
You're right. Right below the heading it says that the data is provided individually for each data provider ("pro Datenlieferant"). I'll keep an eye on that website for new announcements.
